examples/todos/run-offline-docker.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+../docker/run.sh my-custom-heartbeat
```

Comment:
nit: It seems to me that this could be move to docs or to the readme

Reply:
++ I'll remove this